app/services/cart.js

```diff
@@ -124,6 +124,11 @@ export default ApiService.extend(asyncMixin, {
       order_id: order.get("id")
     }).then(data => {
       this.get("store").pushPayload(data);
+      data.orders_packages.map(data => {
+        this.get("store").findRecord("package", data.package_id, {
+          reload: true
+        });
+      });
```

Comment:
Big red flag here, you're creating promises via `findRecord` without returning them or handling failures. That should almost never happen

```suggestion
      return data.orders_packages.map(data => {
        return this.get("store").findRecord("package", data.package_id, {
          reload: true
        });
      });
```

Reply:
sure